account_mtd/models/mtd_hello_world.py

```diff
@@ -0,0 +1,456 @@
+# -*- coding: utf-8 -*-
+
+import requests
+import json
+import logging
+import werkzeug
+import urllib
+
+from odoo import models, fields, api, exceptions
+from datetime import datetime, timedelta
+from urllib.parse import urlparse
+
+
+class MtdHelloWorld(models.Model):
+    _name = 'mtd.hello_world'
+    _description = "Hello world to test connection between Odoo and HMRC"
+    
+    _logger = logging.getLogger(__name__)
```

Comment:
if you make _logger global (put it just under `from urllib.parse import urlparse`) then you can change every instance of `self._logger()` to just `_logger`